src/backend/settings.py

```diff
@@ -137,6 +137,7 @@
             "context_processors": [
                 "django.template.context_processors.debug",
                 "django.template.context_processors.request",
+                "django.template.context_processors.media",
```

Comment:
Why is this context processor needed?
[The docs](https://docs.djangoproject.com/en/2.2/ref/templates/api/#django-template-context-processors-media) say that it only injects the `MEDIA_URL` variable, but this does not seem to be used in the template?
Or are there any other implications?

Reply:
Ups, forgotten to remove that, since I decided to go just with `region.icon.path` as image source, I will remove that.